lib/custom/src/MShop/Service/Provider/Payment/Stripe.php

```diff
@@ -21,6 +22,83 @@ class Stripe
 	extends \Aimeos\MShop\Service\Provider\Payment\OmniPay
 	implements \Aimeos\MShop\Service\Provider\Payment\Iface
 {
+
+	private $beConfig = array(
+		'address' => array(
+			'code' => 'address',
+			'internalcode'=> 'address',
+			'label'=> 'Send address to payment gateway too',
+			'type'=> 'boolean',
+			'internaltype'=> 'boolean',
+			'default'=> '0',
+			'required'=> false,
+		),
+		'authorize' => array(
+			'code' => 'authorize',
+			'internalcode'=> 'authorize',
+			'label'=> 'Authorize payments and capture later',
+			'type'=> 'boolean',
+			'internaltype'=> 'boolean',
+			'default'=> '0',
+			'required'=> false,
+		),
+		'testmode' => array(
+			'code' => 'testmode',
+			'internalcode'=> 'testmode',
+			'label'=> 'Test mode without payments',
+			'type'=> 'boolean',
+			'internaltype'=> 'boolean',
+			'default'=> '0',
+			'required'=> false,
+		),
+		'apiKey' => array(
+			'code' => 'apiKey',
+			'internalcode'=> 'apiKey',
+			'label'=> 'API key',
+			'type'=> 'string',
+			'internaltype'=> 'string',
+			'default'=> '',
+			'required'=> true,
+		),
+		'publishableKey' => array(
+			'code' => 'publishableKey',
+			'internalcode'=> 'publishableKey',
+			'label'=> 'publishable key',
+			'type'=> 'string',
+			'internaltype'=> 'string',
+			'default'=> '',
+			'required'=> true,
+		),
+	);
+
+	protected $feConfig = array(
```

Comment:
Do we need this at all if we use the payment form at the process step?

Reply:
Yes we need it. Without it **process-body-standard.php** is not rendering fields.